<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- No namespace here as we will share this description. 
       Access with slash at the beginning -->

  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find open_base)/urdf/main3.urdf.xacro'" />

  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
     <rosparam file="$(find open_base)/config/joint3_velocity.yaml" command="load" ns="/robot1"/>
      <include file="$(find open_base)/launch/one_robot2.launch" >
       <arg name="init_pose" value="-x 0 -y 2 -z 0" />
       <arg name="robot_name"  value="robot1" />
      </include>
      <node name="robot1_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/robot1"
        args="--namespace=/robot1
        joint_state_controller
        left_joint_velocity_controller
        back_joint_velocity_controller
        right_joint_velocity_controller">
      </node>
      <node
		name="odometry"
		pkg="open_base"
		type="open_base_odometry"
		ns="/robot1"
		output="screen"
		args="pose_cheat">
      </node>
      <node
		name="kinematics"
		pkg="open_base"
		type="open_base_kinematics"
		ns="/robot1"
		output="screen">
      </node>
      <node
		name="encoder"
		pkg="open_base"
		type="open_base_sensor_encoder"
		ns="/robot1/sensor"
		output="screen">
      </node>
      <node
		name="main"
		pkg="open_base"
		type="open_base_main"
		ns="/robot1"
		output="screen">
      </node>
  </group>

  
</launch>
   
